docs/02-app/01-building-your-application/03-rendering/04-edge-and-nodejs-runtimes.mdx

```diff
@@ -37,6 +37,10 @@ The Edge Runtime is ideal if you need to deliver dynamic, personalized content a
 
 For example, code executed in the Edge Runtime [on Vercel cannot exceed between 1 MB and 4 MB](https://vercel.com/docs/concepts/limits/overview#edge-middleware-and-edge-functions-size), this limit includes imported packages, fonts and files, and will vary depending on your deployment infrastructure.
 
+> **Good to know**:
+>
+> - Getting "Module not found: Can't resolve 'fs'" or related errors? This nay be because the Edge Runtime does not support all Node.js APIs like the filesystem. You can use the [Node.js Runtime](#nodejs-runtime) to access these APIs.
```

Comment:
Line 34 basically says this already since Edge Runtime is a “subset” of Node.js runtime. Perhaps we need to clarify that line instead.

Reply:
Thanks @styfle! Updated line 38 and removed lines 40 - 42.